server/daos/tests/products.test.js

```diff
@@ -1,24 +1,24 @@
-const { redis } = require('@server/services/redis');
-const { getCategoryById, getAllCategories } = require('../products');
+// const { redis } = require('@server/services/redis');
```

Comment:
lets remove the commented out stuff. Also if its not needed let's delete the files